force-app/main/default/triggers/Count.trigger

```diff
@@ -0,0 +1,11 @@
+trigger Count on Task (after insert, after update, after delete) {
+     
+    if (trigger.isAfter){
```

Comment:
Poor formatting.

Did you make sure to run `npm install` in your project and set autoformat on save to true in VS Code?

Reply:
No, I'll do it. Thanks!